go-runtime/ftl/map.go

```diff
@@ -0,0 +1,29 @@
+package ftl
+
+import (
+	"context"
+	"sync"
+)
+
+type MapHandle[T, U any] struct {
+	fn     func(context.Context, T) (U, error)
+	getter Handle[T]
+	out    U
+	once   *sync.Once
+}
+
+func (sh *MapHandle[T, U]) Get(ctx context.Context) U {
```

Comment:
ubernit, but should this be `func (mh *MapHandle[T, U])` vs. `func (sh *MapHandle[T, U])`

Reply:
So true!